This code coming from the loop-aes patch breaks the umount-l patch from
Gentoo bug #370051.

Signed-off-by: Lars Wendler (Polynomial-C) <polynomial-c@gentoo.org>
X-Gentoo-Bug: 375165
X-Gentoo-Bug-URL: https://bugs.gentoo.org/375165

--- util-linux-2.19.1/mount/umount.c
+++ util-linux-2.19.1/mount/umount.c
@@ -581,9 +581,6 @@
 
 	file = canonicalize(arg); /* mtab paths are canonicalized */
 
-/* This is too buggy code. If find_loopdev_by_backing_file() actually finds matching */
-/* loop device, and assigns file=loopdev, then writable /etc/mtab case fails horribly */
-#if 0
 	/* if file is a regular file, check if it is associated
 	 * with some loop device
 	 */
@@ -603,7 +600,6 @@
 			break;
 		}
 	}
-#endif
 
 	if (verbose > 1)
 		printf(_("Trying to unmount %s\n"), file);
